### PR TITLE
Add adb pull command to copy measurementsTemp folder from emulator to the project folder.

### PR DESCRIPTION
**Pull request by alex-iotiq**

### CODE REVIEW

**Comment by DaGeRe** [comment]
Thanks for the PR, this looks fine in general.

Just one question: Why do you call `adbPull` in the DependencyManager? As far as I get it, `adbPull`  needs to be called for every test execution, not only when dependencies are updated (since we need to copy data from the emulated machine to the original machine everytime). Therefore, wouldn't it make more sense to move this to `runTest` in the `AnboxTestExecutor`? 

If yes, could you adjust the PR? (Just push a commit to your branch, or change your branch by pushing with `--force`).

**Comment by alex-iotiq** [comment]
Yes this is the better approach. I've updated the PR.

**Comment by alex-iotiq** [comment]
Yes this is the better approach. I've updated the PR.

File: dependency/src/main/java/de/dagere/peass/execution/gradle/AnboxTestExecutor.java [hunk 1]
```diff
@@ -52,7 +71,7 @@ private Process buildGradleProcess(final File moduleFolder, final File logFile,
       String[] anboxOriginals = new String[] { "adb", "shell", "am", "instrument", "-w", "-e", "class" };
```
Comment (DaGeRe) [inline dependency/src/main/java/de/dagere/peass/execution/gradle/AnboxTestExecutor.java:71]: Shouldn't `fetchAdbCall()` be called here as well, instead just using `adb`?

**Comment by DaGeRe** [comment]
Thanks for the adjustment, looks fine.
I added another comment in the code. If this is a correct hint, could you implement it, or otherwise just tell me? Than I'll merge the PR.

File: dependency/src/main/java/de/dagere/peass/execution/gradle/AnboxTestExecutor.java (hunk 1 above)
Comment (alex-iotiq) [reply dependency/src/main/java/de/dagere/peass/execution/gradle/AnboxTestExecutor.java]: It makes sense. I've changed that as well.

**Comment by DaGeRe** [comment]
Looks all file to me now, thanks for adjusting!

**Pull request merged**